app.py

```diff
@@ -0,0 +1,29 @@
+import flask
+import requests
+import os
+
+app = flask.Flask(__name__)
+
+MY_API_SECRET_KEY = "x95xd2dbxc3x175md5"
```

Comment:
It is not advised to display your api secret key in your code as people code misuse it. Instead consider putting in an .env file.